manager/src/main/java/com/redhat/service/smartevents/manager/api/models/responses/ProcessorSchemaEntryResponse.java

```diff
@@ -0,0 +1,50 @@
+package com.redhat.service.smartevents.manager.api.models.responses;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ProcessorSchemaEntryResponse {
+    @JsonProperty("kind")
+    private final String kind = "ProcessorSchemaEntry";
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("type")
+    private String type;
```

Comment:
I guess this should be a `ProcessorType`?

Reply:
On a second thought I would probably agree with @lucamolteni. Given we are referring to them as `source` or `action` elsewhere, we should use the same terminology here.